db/migrate/20150101133905_add_last_login_time_to_users.rb

```diff
@@ -0,0 +1,11 @@
+class AddLastLoginTimeToUsers < ActiveRecord::Migration
+  def self.up
+    add_column :users, :last_sign_in_at, :datetime
```

Comment:
I was wondering, why is the database column called `last_sign_in_at` rather than `last_login_at` (or `last_log_in_at`), since most of the code seems to use `'login` rather than `sign_in`? If there's a reason it has to be that way, that's cool! But if there's not, it would probably be less confusing for future coders if it consistently used one term.